model/modeldecoder/v2/model.go

```diff
@@ -700,14 +700,16 @@ type spanContextDestination struct {
 
 type spanContextDestinationService struct {
 	// Name is the identifier for the destination service,
-	// e.g. 'http://elastic.co', 'elasticsearch', 'rabbitmq'
-	Name nullable.String `json:"name" validate:"required,maxLength=1024"`
+	// e.g. 'http://elastic.co', 'elasticsearch', 'rabbitmq' (
+	// This field is DEPRECATED and will be removed in a future release
```

Comment:
nit: Could you start with the deprecated note so it shows up at the beginning of the Intake API comment?
Similar to https://github.com/elastic/apm-server/blob/f29f10905bde682d833866c0881f282b88c881b3/model/modeldecoder/v2/model.go#L495